.env.example

```
@@ -31,6 +31,72 @@ NEXT_PUBLIC_BACKEND_URL=
 NEXTAUTH_URL=
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
+GITHUB_PACKAGE_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_PACKAGE_TOKEN key should go before the GOOGLE_CLIENT_ID key

(UnorderedKey)